pinentry/src/main/java/me/philio/pinentry/PinEntryView.java

```diff
@@ -316,6 +333,7 @@ private void addViews() {
         editText.setFilters(new InputFilter[]{new InputFilter.LengthFilter(digits)});
         editText.setInputType(inputType);
         editText.setImeOptions(EditorInfo.IME_FLAG_NO_EXTRACT_UI);
+        editText.setPadding(editText.getPaddingLeft(), editText.getPaddingTop(), editText.getPaddingRight(), 100);
```

Comment:
Hi @Philio, good catch; this does look like a mistake. I've removed this change.